manifests/05-version-mapping.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: openshift-console-operator
+  name: version-mapping
+data:
+  "0.0.0_version_console": "docker.io/openshift/origin-console:v4.0"
```

Comment:
Just confirming on your dev cluster these values did get replaced by CVO.

Reply:
So we expect this to be updated to something like `image: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:<hash>`.

Reply:
Commented on the side we could write an e2e test to check this, basically just GET the CM & look for something similar to that string, as a validation that its working.  

I'm not sure if thats the best approach or not. But CVO won't know about it if we just `oc apply -f 8  manifests/05-version-mapping.yaml` in a dev cluster, so its not going to get updated, thus makes it hard to test before merging other than using CI to validate.